File: src/main/java/br/com/jiratorio/domain/response/IssuePeriodResponse.java
package br.com.jiratorio.domain.response;

import br.com.jiratorio.domain.IssuePeriodChart;
import java.util.List;

import br.com.jiratorio.domain.entity.IssuePeriod;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class IssuePeriodResponse {

    private List<IssuePeriod> issuePeriods;

    private IssuePeriodChart issuePeriodChart;

}

File: src/main/java/br/com/jiratorio/domain/response/IssuePeriodResponse.kt
package br.com.jiratorio.domain.response

data class IssuePeriodResponse(
    val id: Long,
    val dates: String,
    val wipAvg: Double,
    val leadTime: Double,
    val avgPctEfficiency: Double,
    val jql: String,
    var issuesCount: Int
)
